tests/system/action/user/test_toggle_presence_by_number.py

```diff
@@ -0,0 +1,125 @@
+from openslides_backend.permissions.management_levels import (
+    CommitteeManagementLevel,
+    OrganizationManagementLevel,
+)
+from openslides_backend.permissions.permissions import Permissions
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserSetPresentActionTest(BaseActionTestCase):
```

Comment:
Please also test the failure cases (too many/no users with a certain number, too many/no users with a certain default number) and edge cases (if a user with a certain number exist and another user has it as the default number, only the first one is toggled; if a user has a normal number and a default number, the default number is ignored, even if it matches)

Reply:
Add some more tests. 3 failure tests and 2 edge tests.